content/millennium/r4/scheduling/appointment.md

```diff
@@ -157,6 +157,7 @@ Patch an existing Appointment.
 
 _Implementation Notes_
 
+* Requires both read and write scopes.
```

Comment:
Is both the read and write scope required for any patch or just when patching a video visit? I think it is just the Video Visit. If so, we need to clarify that. Also, I noticed we use complete sentences instead of fragments.
If so, we could say, 'For Video Visit link patch operations, both read and write scopes are required.

Reply:
@kathypickering  updated here : https://github.com/cerner/fhir.cerner.com/pull/448/commits/d657dae3686d64f189fa7cde96268aa64b28a71b#diff-1d7df8ceb9f0ae8471323516e037c662R160